// bepro-js/build/models/index.js
'use strict';Object.defineProperty(exports, "__esModule", { value: true });exports.OpenerRealFvr = exports.VotingContract = exports.Network = exports.NetworkFactory = exports.ERC20Distribution = exports.ERC721Standard = exports.ERC721Collectibles = exports.ERC20TokenLock = exports.StakingContract = exports.ERC20Contract = undefined;var _ERC20Contract = require('./ERC20/ERC20Contract');var _ERC20Contract2 = _interopRequireDefault(_ERC20Contract);
var _StakingContract = require('./Staking/StakingContract');var _StakingContract2 = _interopRequireDefault(_StakingContract);
var _ERC20TokenLock = require('./ERC20/ERC20TokenLock');var _ERC20TokenLock2 = _interopRequireDefault(_ERC20TokenLock);
var _ERC20Distribution = require('./ERC20/ERC20Distribution');var _ERC20Distribution2 = _interopRequireDefault(_ERC20Distribution);
var _ERC721Collectibles = require('./ERC721/ERC721Collectibles');var _ERC721Collectibles2 = _interopRequireDefault(_ERC721Collectibles);
var _ERC721Standard = require('./ERC721/ERC721Standard');var _ERC721Standard2 = _interopRequireDefault(_ERC721Standard);
var _OpenerRealFvr = require('./custom/RealFvr/OpenerRealFvr');var _OpenerRealFvr2 = _interopRequireDefault(_OpenerRealFvr);
var _Network = require('./BEPRO/Network');var _Network2 = _interopRequireDefault(_Network);
var _NetworkFactory = require('./BEPRO/NetworkFactory');var _NetworkFactory2 = _interopRequireDefault(_NetworkFactory);
var _VotingContract = require('./Voting/VotingContract');var _VotingContract2 = _interopRequireDefault(_VotingContract);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}exports.



ERC20Contract = _ERC20Contract2.default;exports.
StakingContract = _StakingContract2.default;exports.
ERC20TokenLock = _ERC20TokenLock2.default;exports.
ERC721Collectibles = _ERC721Collectibles2.default;exports.
ERC721Standard = _ERC721Standard2.default;exports.
ERC20Distribution = _ERC20Distribution2.default;exports.
NetworkFactory = _NetworkFactory2.default;exports.
Network = _Network2.default;exports.
VotingContract = _VotingContract2.default;exports.
OpenerRealFvr = _OpenerRealFvr2.default;
